browser_patches/firefox-beta/juggler/NetworkObserver.js

```diff
@@ -519,7 +519,8 @@ class NetworkRequest {
     this.httpChannel.QueryInterface(Ci.nsIHttpChannelInternal);
     this.httpChannel.QueryInterface(Ci.nsITimedChannel);
     const timing = {
-      startTime: this.httpChannel.channelCreationTime,
+      issueTime: this.httpChannel.channelCreationTime,
+      startTime: this.httpChannel.asyncOpenTime,
```

Comment:
@pavelfeldman I am not sure if this is correct in FF - https://searchfox.org/mozilla-central/source/netwerk/base/nsITimedChannel.idl#96